src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -35,12 +35,18 @@
             <converters:MarginConverter x:Key="MarginConverter" />
         </Grid.Resources>
         <Grid.RowDefinitions>
+            <RowDefinition Height="30" />
             <RowDefinition Height="*" />
             <RowDefinition Height="5" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
+        <TextBlock Text="{x:Static local:GridOptionPreviewControl.CodeStylePageHeader}" >
```

Comment:
💭 I would have expected the text to be set using the following as a child element, before the hyperlink:

```xml
<Run Text="..."/>
```

❓ Does this need `TextWrapping="Wrap"` for cases where the dialog is not wide enough?

Reply:
done